tests/integ/test_tools.py

```diff
@@ -40,6 +40,10 @@
     vit_image_classification,
     vit_nsfw_classification,
 )
+from vision_agent.tools.tools import (
+    countgd_sam2_video_tracking,
+    owlv2_sam2_video_tracking,
```

Comment:
this is temporary, we'll remove it once we are sage to add it to `FUNCTION_TOOLS`